### PR TITLE
Add func to get first disk

### PR DESCRIPTION
**Pull request by dzhengfy**
This function is to get the first disk on the host or guest, especially
to detect the disk is 'vda' or 'sda'.

Signed-off-by: Dan Zheng <dzheng@redhat.com>

### CODE REVIEW

**Comment by chunfuwen** [comment]
There is similar method in virttest/libvirt_vm.py
```
    def get_first_disk_devices(self):
        """
        Get vm's first disk type block devices.
        """
        disk = {}
        options = "--details"
        result = virsh.domblklist(self.name, options, ignore_status=True,
                                  uri=self.connect_uri)
        blklist = result.stdout_text.strip().splitlines()
        if result.exit_status != 0:
            logging.info("Get vm devices failed.")
        else:
            blklist = blklist[2:]
            linesplit = blklist[0].split(None, 4)
            disk = {'type': linesplit[0],
                    'device': linesplit[1],
                    'target': linesplit[2],
                    'source': linesplit[3]}
        return disk
```

File: virttest/utils_disk.py [hunk 1]
```diff
@@ -1102,6 +1102,23 @@ def get_parts_list(session=None):
     return parts
 
 
+def get_first_disk(session=None):
+    """
+    Get the first disk device on host or guest
+
+    :param session: the vm session
```
Comment (Yingshun) [inline virttest/utils_disk.py:1109]: only support vm session? what about remote host session?

Comment (dzhengfy) [reply virttest/utils_disk.py:1109]: Currently I have not test case to test it if I support remote host. So I would like to enhance it in future if we have the requirement. Thanks.

**Comment by dzhengfy** [comment]
> There is similar method in virttest/libvirt_vm.py
> 
> ```
>     def get_first_disk_devices(self):
>         """
>         Get vm's first disk type block devices.
>         """
>         disk = {}
>         options = "--details"
>         result = virsh.domblklist(self.name, options, ignore_status=True,
>                                   uri=self.connect_uri)
>         blklist = result.stdout_text.strip().splitlines()
>         if result.exit_status != 0:
>             logging.info("Get vm devices failed.")
>         else:
>             blklist = blklist[2:]
>             linesplit = blklist[0].split(None, 4)
>             disk = {'type': linesplit[0],
>                     'device': linesplit[1],
>                     'target': linesplit[2],
>                     'source': linesplit[3]}
>         return disk
> ```

Yes, I notice this function. But sometimes we also need to support both host and guest. That function does not meet the needs. What do you think?

**Review by Yingshun — APPROVED** [review]
LGTM.

File: virttest/utils_disk.py (hunk 1 above)
Comment (Yingshun) [reply virttest/utils_disk.py:1109]: Ok

Comment (Yingshun) [reply virttest/utils_disk.py:1109]: BTW, disks can be listed by avocado.utils.disk.get_disks() on localhost. So I wonder if we can get disks from lsblk command directly: 
for example:
```
json_result = process.run('lsblk --json')
json_data = json.loads(json_result.stdout_text)
[str(disk['name']) for disk in json_data['blockdevices'] if disk['type']=="disk"]
```

**Review by chloerh — APPROVED** [review]

**Review by chunfuwen — APPROVED** [review]
lgtm

**Review by kamilvarga — COMMENTED** [review]
LGTM.

**Pull request merged**